app/components/SearchMapListItem.js

```diff
@@ -11,19 +11,19 @@ class SearchMapListItem extends React.Component {
   }
 
   selectSchool() {
-    SearchActionCreators.selectSchool(this.props.school);
+    SearchActionCreators.selectSchool(this.props.school.location.id);
```

Comment:
I think this action creator should be renamed, as well as the store variable and functions for selectedSchoolId. So their name makes reference to the location id.